src/apps/DDBItemConfig.js

```diff
@@ -37,7 +37,7 @@ export class DDBItemConfig extends FormApplication {
       {
         name: "ignoreItemForChrisPremades",
         isChecked: chris,
-        description: "Ignore this item when replacing effects with those from Chris's Premades module.",
+        description: "Ignore this item when replacing ActiveEffects with those from Chris's Premades module? This is recommended if you use this module and its dependencies aimed at a highly automated game.",
```

Comment:
This appears on in a very niche item configuration dialogue, the recommended language is not correct here, this might imply the user always wants to use this is using Chris' effects, which is not the case.

Reply:
I cannot make sense of your comment.